til-domain/src/main/java/com/til/application/category/CategoryService.java

```diff
@@ -0,0 +1,29 @@
+package com.til.application.category;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.til.domain.category.dto.CategoryDto;
+import com.til.domain.category.model.Category;
+import com.til.domain.category.repository.CategoryRepository;
+
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class CategoryService {
+
+    private final CategoryRepository categoryRepository;
+
+    public List<CategoryDto> getCategoryList() {
+        List<Category> categoryList = categoryRepository.findAll();
+
+        return categoryList.stream()
+            .map(category -> CategoryDto.of(category))
+            .collect(Collectors.toList());
```

Comment:
P3: 이렇게 리팩토링 해보면 좋을 것 같습니다!
```suggestion
        return categoryList.stream()
            .map(CategoryDto::of)
            .collect(Collectors.toList());
```